app/logic/Mail/EWS/EWSAccount.ts

```diff
@@ -198,12 +201,58 @@ export class EWSAccount extends MailAccount {
     return options;
   }
 
+  async checkMaxConcurrency(ex: EWSItemError) {
+    if (ex.type != "ErrorServerBusy") {
+      throw ex;
+    }
+    // Check whether the ConcurrentSyncCalls policy was exceeded.
+    if (/'ConcurrentSyncCalls'/.test(ex.message) && /'(\d+)'/.test(ex.message) && Number(RegExp.$1) < this.maxConcurrency) {
+      this.maxConcurrency = Number(RegExp.$1);
+    }
+    let milliseconds = ex.error.MessageXml?.Value?.Value;
```

Comment:
What makes this a number?
```suggestion
    let milliseconds = parseInt(ex.error.MessageXml?.Value?.Value);
```